src/test/java/com/google/cloud/pontem/SpannerUtilTest.java

```diff
@@ -330,12 +331,24 @@ public void testGetSpannerType() {
   }
 
   @Test(expected = RuntimeException.class)
-  public void testGetSpannerType_error1() {
+  public void testGetSpannerType_error0() {
     SpannerUtil.getSpannerType("ARRAY<BTES(1024)>");
   }
 
+  @Test(expected = RuntimeException.class)
+  public void testGetSpannerType_error1() {
+    SpannerUtil.getSpannerType("ARRAY<BTES@#@(1024)>");
```

Comment:
Given that both types are BTES instead of BYTES is there really a difference between case 0 and 1? Or should case 1 be using BYTES and fail on @#@ ?